docs-ref-conceptual/index.yml

```diff
@@ -243,6 +243,11 @@ sections:
       image:
         src: https://docs.microsoft.com/media/logos/logo_vs-team-services.svg
       title: VSTS
+    - title: GitHub Actions 
```

Comment:
There are two `title`s here- I'm not familiar with how this particular YAML file is used but I thought I would call it out because it seems unusual to me.

Thanks!

Reply:
YAML can be a bit confusing but there's an `-` which marks an new section, and this new section starts with the title attribute while the other ends with it, so I think it's good. Thanks for the sharp eyes!